README.md

```diff
@@ -59,6 +59,59 @@ console.log(results1.labels); // [ 3, 0, 1, 2 ]
 console.log(results1.distances); // [ 0, 1, 1, 4 ]
 ```
 
+## Setup for contibutors
```

Comment:
I recommend putting this part in a new CONTRIBUTING.md as there are also Windows and Linux related information (I can add it later).